_data/navbars/security.yml

```diff
@@ -86,3 +86,5 @@ section:
   section:
   - title: Network visibility
     path: /security/calico-enterprise/network-visibility
+  - title: Advanced egress access controls with Calico Enterprise
+    path: /security/calico-enterprise/egress-access-controls
```

Comment:
I think this is missing `/security/calico-enterprise/compliance`

Reply:
That is expected; the doc wasn't ready. However, in my review of https://github.com/projectcalico/calico/pull/3170 I missed that while the nav bar entry was removed the doc and accompanying images were not.

I'll remove the compliance doc and images from master and add that commit to this PR.

Reply:
Fixed in b8302d7

Reply:
I put compliance doc in, then Alex said it was not ready/reviewed so I removed it from the TOC. This PR is only for Advanced egress.